clients/admin-ui/src/features/common/ClipboardButton.tsx

```diff
@@ -80,15 +55,11 @@ const ClipboardButton = ({ copyText, ...props }: ClipboardButtonProps) => {
     >
       <Button
         icon={<CopyIcon />}
-        color={iconColor}
```

Comment:
after upgrading Ant, there were TS errors on this button. I took a moment to clean it all up since it's manually doing all of the button styling that we don't need anymore.